charts/nomad-bridge/templates/config/serviceaccount.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.serviceAccount.create -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "agent.serviceAccountName" . }}
```

Comment:
Don't you also need to assign the ServiceAccount to your Pod through the StatefulSets?

Reply:
Added